src/css/_typography.scss

```diff
@@ -1,5 +1,5 @@
 // You can still use import to import a stylesheet from Google Fonts.
-@import url("https://fonts.googleapis.com/css2?family=Lobster&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Oleo+Script&display=swap");
 @import url("https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@200&=swap");
```

Comment:
See if you can combine these in Google Fonts by selecting them in the same download. That way, you aren't making two requests out to Google Web fonts.